src/tpm2_pytss/constants.py

```diff
@@ -229,6 +233,66 @@ def _fix_const_type(cls):
             TPM_FRIENDLY_INT._copy_and_set(cls, sc)
         return cls
 
+    def marshal(self):
+        """Marshal instance into bytes.
+
+        Returns:
+            Returns the marshaled type as bytes.
+        """
+
+        # Not all constants should be marshaled.
+        no_seralize_msg = getattr(self, "_no_marshal_msg", None)
+        if no_seralize_msg:
+            raise RuntimeError(f"{no_seralize_msg}")
```

Comment:
> perhaps `no_marshal_msg` instead of `no_seralize_msg` and use NotImplementedError instead of RuntimeError. Or simply override the method in ESYS_TR

I like overriding it in ESYS_TR, much simpler.